src/main/java/org/tron/common/overlay/discover/NodeManager.java

```diff
@@ -64,7 +64,7 @@
   Consumer<DiscoveryEvent> messageSender;
 
   NodeTable table;
-  private Map<String, NodeHandler> nodeHandlerMap = new ConcurrentHashMap<>();
+  public Map<String, NodeHandler> nodeHandlerMap = new ConcurrentHashMap<>();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make nodeHandlerMap a static final constant or non-public and provide accessors if needed. ([squid:ClassVariableVisibilityCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AClassVariableVisibilityCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=872&issues=AWOrsapZ_Tl7jhO80k-M&open=AWOrsapZ_Tl7jhO80k-M)

[ISSUE_KEY]: AWOrsapZ_Tl7jhO80k-M